setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='django-two-factor-auth',
-    version='1.15.5',
+    version='1.16',
```

Comment:
Can you remove the commits that change the version string? This is something that's done at release time.